src/main/java/exercise31/KarvonenHeartRate.java

```diff
@@ -13,19 +14,24 @@ public int formula(int age, int restingHR, double intensity) {
         return bpm;
     }
 
+    //Naming / data-type?
     public String[] checkDifferentIntensities(int age, int restingHR) {
+        //Shorter way to do addition?
         for (double intensity = 55; intensity <= 95; intensity = intensity + 5) {
```

Comment:
Can you increment `intensity` more concisely?  Also it's odd to use a double for the loop counter then only store whole numbers in it.